### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang:1.18 as builder
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Please rename from fuzz.libfuzzer.

File: mayhem/fuzz.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package fuzz
```
Comment (xansec) [inline mayhem/fuzz.go]: Please rename to something more descriptive.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

File: mayhem/fuzznjson.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package fuzznjson
```
Comment (xansec) [inline mayhem/fuzznjson.go:1]: I would prefer something like "fuzzUnmarshal", for future reference. I'll approve this for now.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**